docs/guides/ha-cluster/fence_nnf.py

```diff
@@ -0,0 +1,217 @@
+#!@PYTHON@
+
+# Fence agent for Near Node Flash devices
+
+import sys
+import logging
+import atexit
+
+sys.path.append("@FENCEAGENTSLIBDIR@")
+
+from fencing import *
+from fencing import fail, fail_usage, run_delay
+
+try:
+    from kubernetes import client
+except ImportError:
+    logging.error("Couldn't import kubernetes.client - not found or not accessible")
```

Comment:
Why handle this one specifically?

Reply:
Because it has to be manually installed using pip